packages/connector-wechat-native/src/index.ts

```diff
@@ -107,6 +94,7 @@ export default class WeChatNativeConnector implements SocialConnector {
       // Response properties of user info can be separated into two groups: (1) {unionid, headimgurl, nickname}, (2) {errcode, errmsg}.
       // These two groups are mutually exclusive: if group (1) is not empty, group (2) should be empty and vice versa.
       // 'errmsg' and 'errcode' turn to non-empty values or empty values at the same time. Hence, if 'errmsg' is non-empty then 'errcode' should be non-empty.
+      // 'openid' is required but the response of getUserInfo is consistent as long as access_token is valid. In other words, 'openid' can be an arbitrary non-empty string, but it makes no sense that 'openid' can be some other strings other than its own value.
```

Comment:
not following this comment.  `openId` can be any string?  and Wechat identifies the requested userInfo by access_token only?

Reply:
I've tried to call WeChat API with lots of different inputs to elaborate errors in different cases.
`openid` can be any non-empty string as long as `access_token` is valid, and it should not be undefined, null or empty string.